### PR TITLE
readme changed

### PR DESCRIPTION
**Pull request by ovichowdhury**
readme updated for own version

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1 @@
...
-- [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
-
-You can check out [the Next.js GitHub repository](https://github.com/vercel/next.js/) - your feedback and contributions are welcome!
-
-## Deploy on Vercel
-
-The easiest way to deploy your Next.js app is to use the [Vercel Platform](https://vercel.com/new?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) from the creators of Next.js.
-
-Check out our [Next.js deployment documentation](https://nextjs.org/docs/deployment) for more details.
+This is my readme
```
Comment (ovichowdhury) [inline README.md]: why ?

File: README.md [hunk 2]
```diff
@@ -1,34 +1 @@
...
-## Getting Started
-
-First, run the development server:
-
-```bash
-npm run dev
-# or
-yarn dev
-```
-
```
Comment (ovichowdhury) [inline README.md:12]: important

**Review by ovichowdhury — COMMENTED** [review]
check

File: README.md [hunk 3]
```diff
@@ -1,34 +1 @@
...
-- [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
-- [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
-
-You can check out [the Next.js GitHub repository](https://github.com/vercel/next.js/) - your feedback and contributions are welcome!
-
-## Deploy on Vercel
-
-The easiest way to deploy your Next.js app is to use the [Vercel Platform](https://vercel.com/new?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) from the creators of Next.js.
-
-Check out our [Next.js deployment documentation](https://nextjs.org/docs/deployment) for more details.
```
Comment (codesafin) [inline README.md]: Keep it

Comment (ovichowdhury) [reply README.md]: ok

Comment (codesafin) [reply README.md]: are you done?

Comment (ovichowdhury) [reply README.md]: yes

**Review by codesafin — APPROVED** [review]

**Pull request merged**